tasks/maintain-a-collection-of-new-products.json

```diff
@@ -9,8 +9,8 @@
     "mechanic/user/trigger",
     "mechanic/scheduler/daily"
   ],
-  "subscriptions_template": "mechanic/user/trigger\r\nmechanic/scheduler/daily",
```

Comment:
lol thank you - I was badly confused by [the diff up above](https://github.com/lightward/mechanic-tasks/pull/46/files#diff-93dd54f83ff19d26417d692a55fa65fa4447fba172a5aca9e3dd74c804bd391bL26-R26), but _this_ makes sense :D